docs/modules/sql/pages/mapping-to-mongo.adoc

```diff
@@ -165,9 +164,77 @@ DATA CONNECTION myMongo
 ----
 Notice that there is no mention of `TYPE MONGO` this time; it’s automatically assumed by the SQL engine when you provide MongoDB data connection. This works with both schema provided or not.
 
+
+[NOTE]
+====
+Specify the database name using one of the following options:
+
+* Add `OPTIONS ('database' = 'myDatabase')` in `CREATE DATA CONNECTION`
+* Add `OPTIONS ('database' = 'myDatabase')` in `CREATE MAPPING`
+* Use the full external name, e.g., `CREATE MAPPING people EXTERNAL NAME myDatabase.people (...)`
+====
+
+== Supported Object Types
+
+There may be one or more object types for a connector. The Mongo SQL connector has two object types:
+
+ - `Collection`: Represents a batch read from a given collection. This is the default object type.
+ - `ChangeStream`: Represents reading a stream of events.
+
+To change the object type, append `OBJECT TYPE X` after `DATA CONNECTION` / `TYPE`. For example:
+
+[tabs]
```

Comment:
The tabs are not displaying in the preview as the syntax is slightly wrong. Try this instead:

```[tabs]
====
Using Data Connection::
+
--
[source,sql]
----
CREATE MAPPING people (
    firstName VARCHAR(100),
    lastName VARCHAR(100),
    age INT
)
DATA CONNECTION myMongo
OBJECT TYPE ChangeStream
OPTIONS (...)
----
--
Using TYPE::
+
--
[source,sql]
----
CREATE MAPPING people (
firstName VARCHAR(100),
lastName VARCHAR(100),
age INT
)
TYPE Mongo
OBJECT TYPE ChangeStream
OPTIONS (...)
----
--
====

Reply:
Done: https://github.com/hazelcast/hz-docs/pull/761/commits/36b3c1aeb14ce3e00923f86ca9f95dff8ac1f68a